test/check-cli

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+
+import composertest
+
+
+class TestCli(composertest.ComposerTestCase):
+    def runCliTest(self, script):
+        r = self.execute(["CLI=/usr/bin/composer-cli", "/tests/test_cli.sh", script])
+        self.assertEqual(r.returncode, 0)
+
+    def testBlueprintsSanity(self):
+        self.runCliTest("/tests/cli/test_blueprints_sanity.sh")
+
+    def testComposeSanity(self):
+        self.runCliTest("/tests/cli/test_compose_sanity.sh")
+
+
+class TestImages(TestCli):
+    def testExt4(self):
+        self.runCliTest("/tests/cli/test_compose_ext4-filesystem.sh")
+
+    def testPartitionedDisk(self):
+        self.runCliTest("/tests/cli/test_compose_partitioned-disk.sh")
+
+    def testTar(self):
+        self.runCliTest("/tests/cli/test_compose_tar.sh")
+
+    def testQcow2(self):
+        self.runCliTest("/tests/cli/test_compose_qcow2.sh")
+
+    def testLiveIso(self):
+        self.runCliTest("/tests/cli/test_compose_live-iso.sh")
+
+
+class TestCloud(TestCli):
```

Comment:
Just a note at this point:
here all of the test methods will be executed in sequence and uploads (especially for Azure) tend to take a lot of time. We want to execute all of the cloud methods in parallel, and have them report separate statuses via the GitHub API. This we can deal with later.

Execution wise all of these need a VM just to build the image and then everything else happens on the cloud vendor side.

Reply:
Yeah... I've moved them into a separate file and made the test being run dependent on `TEST_SCENARIO`, which is set by Cockpit's test framework to get separate GitHub statusses.